lib/src/components/NxSmallVulnerabilityCounter/NxSmallVulnerabilityCounter.scss

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use "sass:math";
+@use "../../scss-shared/nx-text-helpers";
+
+@mixin counter-colors {
+  @each $severity in critical, high, medium, low, none {
+    &--#{$severity} {
+      background-color: var(--nx-color-vulnerability-#{$severity});
```

Comment:
The rendered color here doesn't match the spec, at least for critical (I haven't checked the others yet).  Are the color variables correct?  Is this dependent on the changes in https://github.com/sonatype/sonatype-react-shared-components/pull/1197 ?

Reply:
> The rendered color here doesn't match the spec, at least for critical (I haven't checked the others yet). Are the color variables correct? Is this dependent on the changes in #1197 ?

Hmm, according to Kiran and Jason, the background color should match the threat counter colors.
So this was a direct mapping to that. I'm guessing, the mock color are based on changes in https://github.com/sonatype/sonatype-react-shared-components/pull/1197? I will double check with Kiran today.

Reply:
> Hmm, according to Kiran and Jason, the background color should match the threat counter colors.
So this was a direct mapping to that. I'm guessing, the mock color are based on changes in https://github.com/sonatype/sonatype-react-shared-components/pull/1197? I will double check with Kiran today.

1197 is merged.  I recommend you merge 12.x-backports into your feature branch and then that into this branch, and then update your visual teset snapshots.

Reply:
> > Hmm, according to Kiran and Jason, the background color should match the threat counter colors.
> > So this was a direct mapping to that. I'm guessing, the mock color are based on changes in #1197? I will double check with Kiran today.
> 
> 1197 is merged. I recommend you merge 12.x-backports into your feature branch and then that into this branch, and then update your visual test snapshots.

Visual tests are updated here: [c348095](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/c3480950edba5ad98bb0f6dee79ead499dee9c35)